man/systemd.network.xml

```diff
@@ -737,7 +737,7 @@ Table=1234</programlisting></para>
           may trigger the start of the DHCPv6 client if the relevant flags are set in the RA data, or
           if no routers are found on the link. The default is to disable RA reception for bridge
           devices or when IP forwarding is enabled, and to enable it otherwise. Cannot be enabled on
-          bond devices and when link-local addressing is disabled.</para>
+          devices aggregated in a bond device and when link-local addressing is disabled.</para>
```

Comment:
Not your fault, but "and" -> "or" ??
`aggregated in a bond device or when link-local addressing is disabled`

Reply:
Done.

Reply:
Thank you!